src/formatDate.js

```diff
@@ -51,6 +51,32 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const object = {};
+  const arrDate = date.split(fromFormat[3]);
+
+  for (let i = 0; i < arrDate.length; i++) {
+    if (fromFormat[i] === 'YY') {
+      if (+arrDate[i] < 30) {
```

Comment:
Don't use "magic values". Create `const` values with descriptive names for such values